feat: Print action outputs to console (#2265)
dagger/dagger: docs/guides/1228-handling-outputs.md

[✔] actions.deploy.container.script
[✔] client.filesystem."./".read
[✔] actions.deploy
[✔] actions.test
[✔] actions.build.run
[✔] actions.build.contents
[✔] actions.deploy.container
[✔] client.filesystem."./_build".write
[✔] actions.deploy.container.export
Field      Value
url        "https://dagger-todo.netlify.app"
deployUrl  "https://62698983ffe8661d60613431--dagger-todo.netlify.app"
logsUrl    "https://app.netlify.com/sites/dagger-todo/deploys/62698983ffe8661d60613431"

```

This is very useful to get immediate feedback on an action's results.

## Piping a result

Besides the `plain` format (the default), you can also use `json` or `yaml`. JSON is particularly useful if you want to pipe a result into another process:

```cue file=../tests/guides/handling-outputs/default.cue
```

```shell
➜ dagger --output-format json do push | jq '.result'
"localhost:5042/alpine:latest@sha256:a777c9c66ba177ccfea23f2a216ff6721e78a662cd17019488c417135299cd89"
```

:::tip
You can silence the `info` logs by raising the log level (or redirecting _stderr_ somewhere else):

```shell
➜ dagger -l error --output-format json do push | jq '.result'
```

:::

## Saving into a file

> You can also save the output to a file, even without a shell, with the `--output` flag. Let's do it in _yaml_ this time:

Reviewer: nit: "even without a shell"  seems unnecessary to clarify

```suggestion
You can also save the output to a file using the `--output` flag. Let's do it in _yaml_ this time:
```
